components/layouts/measures-layout.js

```diff
@@ -461,6 +461,7 @@ export default function MeasuresLayout(props) {
                           numberOfR6,
                         ]}
                         handleFilters={(checkboxState) => handleFilters(checkboxState, 'r_ladder')}
+                        r_ladderStyleProp='r_ladderCSSClasses'
```

Comment:
very very small thing but should just call it `r_ladderStyle` as it is already implied it is a prop :) and same for other one

Reply:
also thinking about how we are now using camel case and underscore mixed in places, thats my bad, will go around and fix it!